### ISSUE TITLE
Compounds sent for testing in Dundee, December 2017

### ISSUE THREAD

**Issue by david1597** [OpenSourceMalaria/Series4#10]
The final batch of compounds for 2017 have been shipped to Dundee for testing, including:

- a caged compound (OpenSourceMalaria/OSM_To_Do_List#510)

- the Pfizer biosynthesis compound (OpenSourceMalaria/OSM_To_Do_List#525), and an H-bond donor analogue

- late stage functionalisation compounds

- Suzuki coupling compounds

![decdundee](https://user-images.githubusercontent.com/30584472/33532086-48ece5e2-d8ea-11e7-8948-aac92558d82c.jpg)

**Comment by mcoster** [comment]
Nice work @david1597 - I particularly like the late-stage functionalisation compounds. We recently ordered a bunch of the sulfinate reagents in for a different project, and I was going to get someone to try that chemistry on some OSM compounds - looks like you've already gone down that path and it works! :) Out of interest, did you see any products of reaction at other positions?

**Comment by david1597** [comment]
Yep we're keen to see how these all turn out, @mcoster. All those late stage functionalisation compounds were the work of @ConorGraham in his Honours project over the last year. He didn't see any products from reaction at the other positions. It's interesting yep! That sulfonamide above is from @maratsydney series. I'm also considering installing it in place of the phenol in the Pfizer biosynthesis compound.

**Comment by mattodd** [comment]
Yes, sorry about the work from @ConorGraham not being public domain yet - the application of methods to OSM compounds evolved during the year from what is otherwise a closed project. He's just finishing up Hons, but then we'll get all the relevant lab book entries public domain and summarise some things.

**Comment by mcoster** [comment]
@david1597 - a sulfonamide in place of the phenol in the biosynthesis series sounds like a great idea. If it's successful, you can also easily acylate sulfonamides and the product _N_-acylsulfonamides would be also be an interesting bioisostere.

**Comment by MFernflower** [comment]
I'm curious if right-hand-para-sulfonamides have ever been tested - I know 4-methylsulfonyl was tested but was a sulfonamide ever tested/biosynthesis assayed ? 

![eeee](https://user-images.githubusercontent.com/3164942/33587382-df08001a-d93b-11e7-95e2-008486b5576c.png)

**Comment by david1597** [comment]
Results are in. There are a couple of issues... 1) The potency of the synthesised 'Pfizer phenol' is not as expected. The results from the biosynthesis had indicated this to be single digit nM, we've got a result of over 4 uM here. This needs to be investigated asap when I return in January, starting with checking the  spectra against that obtained from the biosynthesis (we need to obtain these first). Obviously, this is is the basis of many of our current efforts. 2) Some of the repeated runs aren't in agreement. Both values are shown in these cases, and with a *. Will investigate this too when everyone returns.

Merry Christmas everyone! Here's to a happy, healthy, and successful 2018!

![decdundee](https://user-images.githubusercontent.com/30584472/34318220-ad0e4328-e816-11e7-9bc6-b795c58935b0.jpg)

**Comment by MFernflower** [comment]
the fact that MMV1577581 comes back as mildly potent makes we wonder if we should consider making and screening the acetoxy and methoxy derivatives of this compound?

**Comment by drc007** [comment]
Hi,

There are two structures numbered

MMV1577581 

One on top row, the other on the bottom.

Chris

**Comment by edwintse** [comment]
@drc007 Thanks for pointing that out. The top row compound has the correct MMV number. The bottom row compound should be **MMV1577580**

**Comment by edwintse** [comment]
@MFernflower The methoxy derivative has been tested before (MMV688897) and is inactive at >10 uM. Not sure the acetoxy is worth pursuing though.

**Comment by drc007** [comment]
I’m not sure how stable the acetoxy would be, very susceptible to hydrolysis.

**Comment by MFernflower** [comment]
Seems like the hydroxy compound is a dead end lead? Any idea why the phenolic compound has a different potency run to run?

**Comment by MFernflower** [comment]
Would it be possible to couple MMV1577580 to some N,N-Dimethylsulfamoyl chloride ? sulfamates show interesting hydrogen bonding properties

**Comment by mattodd** [comment]
Merry Christmas everyone.
Just on MMV1557865 (even though it's the holidays I can't help thinking about it) - the disconnect between the compound arising from the Pfizer work and the chemical resynthesis is the top priority for early Jan, when we resume. It's too early to abandon this since we need to check what we had in both cases, meaning we need to compare the NMR spectra and any other data to confirm identities. Along that general line I like the look of MMV1577574, with the H-bond donor in the same general position, but one Ar ring fewer.

**Comment by edwintse** [comment]
UPDATE:
Hi everyone, it appears that there's been a mix up with a few of the MMV numbers and associated potency data. (You'll notice that there are two MMV1577574's. The first should in fact be MMV1577573.)
**_As a result the potency data for 573 to 581 are not for the respective compounds_**.
The remaining compounds are ok. Everything should be resolved in early January. We'll post again when everything is fixed.

**Comment by cdsouthan** [comment]
To err is human and no probs of course.  But JFTR did you prepare the shipment on your own rather than arranging for two of you to cross-check each other?

**Comment by edwintse** [comment]
Previously I have done the shipments but for this last batch I was already away on holiday with limited internet access so David organised the shipping. The registration of the compounds was fine. There was just a mix up with the spreadsheet that was sent to Dundee. We will cross-check for future shipments.

**Comment by david1597** [comment]
Corrected potency results for the first run of these compounds is here:

![decdundee](https://user-images.githubusercontent.com/30584472/34809553-c88eed96-f6e9-11e7-8871-dd07a7e01cba.jpg)

**Comment by MFernflower** [comment]
Is it possible that some of the Pfizer phenol compound oxidized to the
carboxylic acid  - thus explaining the differences in potency between runs?

**Comment by david1597** [comment]
The second run of data has finally come through from the December shipment. Averaged potencies are indicated below.

- it was a largely non-potent selection
- OSM-S-412, the 'Pfizer phenol' is confirmed multimicromolar potency, although we now know this was not the potent compound originally made in that biosynthesis.
- one of Ed's cages, OSM-S-433 is on the threshold of sub micromolar potency
- the solitary potent hit OSM-S-369 from this batch was a repeat synthesis. This batch gave a value of 301 nM compared to the previous value of 251 nM. 

The data have:
- been entered into the Master Chemical List
- uploaded to the '[Not yet Incorporated](https://github.com/OpenSourceMalaria/Series4/wiki/Biological-Data-currently-not-incorporated-into-the-main-Wiki-sections)' section of the wiki
- uploaded to my [ELN](https://au-mynotebook.labarchives.com/share/David%2520Smith%2520OSM/MTc2Ljh8MzAxNzEvMTM2L1RyZWVOb2RlLzI4NTM1NTQ3OTB8NDQ4Ljc5OTk5OTk5OTk5OTk1), along with the raw data curves


![dundee december results](https://user-images.githubusercontent.com/30584472/40593606-edc5b42a-626c-11e8-8fed-93d95bf914af.jpg)

There are still some remaining action points:
- revision of OSM-S-328 structure in ScienceCloud. The structure shown here is the corrected revised structure.
- correction of OSM-S-412 data as it still has the original Pfizer potencies associated with this number. We now know that these results correspond to a different structure.
